.azure/pipelines/jobs/default-build.yml

```
@@ -120,11 +120,11 @@ jobs:
       ${{ if eq(parameters.agentOs, 'Windows') }}:
         ${{ if eq(variables['System.TeamProject'], 'public') }}:
           name: NetCore1ESPool-Public
-          demands: ImageOverride -equals 1es-windows-2019-open
+          demands: ImageOverride -equals 1es-windows-2022-open
```

In hopes the other image is a bit more complete
```suggestion
          demands: ImageOverride -equals build.windows.amd64.vs2022.open
```